### PR TITLE
Aarch64 build fixes

### PR DESCRIPTION
**Pull request by Karry**
...and update build instructions

### CODE REVIEW

File: anthy/README.md [hunk 1]
```diff
@@ -1,15 +1,27 @@
...
-7. `exit`
-8. `mb2 -t SailfishOS-* -s ../anthy.spec build`
+## Build Anthy with Sailfish SDK
+
+```bash
+# configure SDK
+PATH=$PATH:~/SailfishOS/bin/
+
+export OS_VERSION=4.1.0.24
+export ARCHITECTURE=aarch64
```
Comment (topiasv) [inline anthy/README.md:15]: This wouldn't work on a 32-bit arm architecture, right? Should probably add separate build instructions for those as well.

Comment (Karry) [reply anthy/README.md:15]: Of course. these variables (PATH, OS_VERSION, ARCHITECTURE) have to be initialized regarding your installation and target platform. I added comments to them...

**Comment by Karry** [comment]
Hi @topiasv , is there missing something?

File: anthy/README.md (hunk 1 above)
Comment (knokmki612) [reply anthy/README.md:15]: > I added comments to them...

Looks good

**Review by knokmki612 — APPROVED** [review]
I'm sorry for extremely slow response.
I confirmed this instruction is valid for Sailfish SDK 3.x
by the way,[ the known issue](https://github.com/sfos-ja/libanthy-qml-plugin/tree/master/anthy#known-issue) is still occurred in my environment.
I may change the instruction to make easier for me.

anyway, I would like to prioritize that merge this pull-request because this change is better than current.

Thank you for your contribution 🙂

**Pull request merged**